lib/subscription-connection.js

```diff
@@ -35,6 +35,7 @@ module.exports = class SubscriptionConnection {
     this.onConnect = onConnect
     this.subscriptionContexts = new Map()
     this.context = context
+    this.isReady = false
```

Comment:
where is `isReady = true` set?

Reply:
at the end of handleConnectionInit, it's to verify the message order as @PacoDu mentioned